liblangutil/SourceReferenceFormatter.cpp

```diff
@@ -70,6 +70,16 @@ AnsiColorized SourceReferenceFormatter::errorColored() const
 	return AnsiColorized(m_stream, m_colored, {BOLD, RED});
 }
 
+AnsiColorized SourceReferenceFormatter::warningColored() const
+{
+	return AnsiColorized(m_stream, m_colored, {BOLD, YELLOW});
+}
+
+AnsiColorized SourceReferenceFormatter::infoColored() const
+{
+	return AnsiColorized(m_stream, m_colored, {BOLD, YELLOW});
```

Comment:
Why not `WHITE`? Is that because you think it looks bad or is there some technical obstacle?

Reply:
The motivation was : In case we've white background then it would create difficulty. But as suggested by you I've changed it to white.

Reply:
Could you check how it looks like on white background then? To be more specific, we want to use the default foreground color. I think that terminals with white background will have it remapped to black or something else, but we should verify this.

Reply:
What about this? Have you checked it how it works in a white terminal?

Reply:
Greetings of the day @cameel,

As of now I haven't checked it on white terminal.

Thanks & Regards